example_2/ros2_control_demo_example_2.xml

```diff
@@ -0,0 +1,9 @@
+<library path="ros2_control_demo_example_2">
```

Comment:
Please delete other plugins from the `xml` file in the previous package. Just to keep better overview.